DirectProgramming/C++SYCL_FPGA/Tutorials/Features/task_sequences/naive/src/main.cpp

```diff
@@ -0,0 +1,170 @@
+#include <iostream>
+
+// oneAPI headers
+#include <sycl/ext/intel/experimental/pipes.hpp>
+#include <sycl/ext/intel/experimental/task_sequence.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/sycl.hpp>
+
+#include "exception_handler.hpp"
+
+constexpr int kVectSize = 128;
+
+// Forward declare the kernel names in the global scope.
+// This FPGA best practice reduces name mangling in the optimization reports.
+class IDNaive;
+
+using ValueT = int;
+
+// Minimum capacity of a pipe.
+// Set to 0 to let compiler decides on the pipe capacity.
```

Comment:
```suggestion
Set to 0 to allow the compiler to save area if possible.
```

I thought the compiler always decides the pipe capacity, regardless of what you specify? choosing `0` just sets a lower bound.